tests/unittests/tests-rtc/tests-rtc.c

```diff
@@ -63,7 +63,7 @@ static void test_rtc_sec_wrap(void)
         .tm_year = 100,
         .tm_wday =   0,
         .tm_yday =   0,
-        .tm_isdst=   1
+        .tm_isdst=  -1,
```

Comment:
Should we also set this for the tests where `.tm_isdst = 0`?

Reply:
Then the tests fail in a different way...

```
READY
s
START
.
rtc_tests.test_rtc_compat (tests/unittests/tests-rtc/tests-rtc.c 25) exp 14 was 13
.
rtc_tests.test_rtc_sec_wrap (tests/unittests/tests-rtc/tests-rtc.c 25) exp 1 was 0
.......
run 9 failures 2
```


I guess because those dates are within some DSTs?

Reply:
(this was from a local run, to be clear)